doc/flame/rendering/images.md

```diff
@@ -120,6 +120,32 @@ class MyGame extends Game {
 ```
 
 
+## Loading images from the network
```

Comment:
I think this section should be further down in the document? Feels like Sprite for example is of higher priority?